cscs-checks/analytics/spec-accel/spec.py

```diff
@@ -0,0 +1,93 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+from reframe.core.launchers.registry import getlauncher
+
+@rfm.simple_test
+class SpecAccelCheck(rfm.RegressionTest):
+    def __init__(self):
+        super().__init__()
+        self.descr = 'SPEC-accel benchmark'
```

Comment:
Remove the setting of `self.descr` the default one is fine.

Reply:
The default is the same as the name, which is not so human readable. So, you may keep the current description.